pkg/reconciler/ksvc/resources/knative_route_test.go

```diff
@@ -60,13 +61,11 @@ func TestKnativeRoute(t *testing.T) {
 							RuntimeVersion: "1.13",
 						},
 					},
-					Canary: &v1alpha1.CanarySpec{
-						TrafficPercent: 20,
-						ModelSpec: v1alpha1.ModelSpec{
-							Tensorflow: &v1alpha1.TensorflowSpec{
-								ModelURI:       "s3://test/mnist-2/export",
-								RuntimeVersion: "1.13",
-							},
+					CanaryTrafficPercent: 20,
+					Canary: &v1alpha1.ModelSpec{
+						Tensorflow: &v1alpha1.TensorflowSpec{
+							ModelURI:       "s3://test/mnist-2/export",
```

Comment:
Is this ModelURI valid?

Reply:
this unit test just test `CreateKnativeRoute` function whether KFService creates the expected Route resource, so model URI does not need to be valid since it does not actually load a model.